deepchem/models/torch_models/layers.py

```diff
@@ -2972,3 +2972,65 @@ def _dynamic_partition(self, input_tensor: torch.Tensor,
         partitions = [input_tensor[mask] for mask in partition_masks]
 
         return partitions
+
+
+class DTNNEmbedding(nn.Module):
+    """DTNNEmbedding layer for DTNN model.
+    Assign initial atomic descriptors.
```

Comment:
Should include a reference to the original work describing DTNN

Reply:
Done